doc/openocd.texi

```diff
@@ -11524,6 +11524,20 @@ as in the mie CSR (defined in the RISC-V Privileged Spec).
 For details on this trigger type, see the RISC-V Debug Specification.
 @end deffn
 
+@deffn {Command} {riscv reserve_trigger} [index [@option{on|off}]]
+Manages the set of reserved triggers. Reserving a trigger results in OpenOCD
+not using it internally (e.g. skipping it when setting a watchpoint or a
+hardware breakpoint), so that the user or the application being run has
```

Comment:
Detail: "being run" can be dropped which makes the sentence easier to read and no meaning is lost.

```suggestion
hardware breakpoint), so that the user or the application has
```

Reply:
Addressed.